Library/Formula/pius.rb

```diff
@@ -10,7 +11,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     # Replace hardcoded gpg path (WONTFIX):
     # https://sourceforge.net/p/pgpius/bugs/12/
-    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
+    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
```

Comment:
I can update the PR to include that. Just an FYI that I will be without internet for a week so unable to respond to additional requests till then should they come up. 

Sent from my iPhone

> On Oct 11, 2015, at 11:12 AM, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/pius.rb:
> 
> > @@ -10,7 +11,7 @@ def install
> >      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
> >      # Replace hardcoded gpg path (WONTFIX):
> >      # https://sourceforge.net/p/pgpius/bugs/12/
> > -    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
> > +    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
> It's a special dependency that can be satisfied by having either gpg or gpg2 in the $PATH.
> 
> I guess if gpg2 becomes the default soon I'm fine with this as-is. Perhaps it's worth updating the comment above the inreplace to reference this discussion.
> 
> —
> Reply to this email directly or view it on GitHub.
>